gdk/commands/Command.py

```diff
@@ -91,6 +90,14 @@ def _non_conflicting_args_map(self):
                     _non_conflicting_args_map[c_arg] = c_arg_set
         return _non_conflicting_args_map
 
+    def get_sub_c(self, command, sub_c):
```

Comment:
Could we use types?. command seems to be a string right that we get from a dictionary and sub_c is what? Whatever cli_tool.cli_model returns, which is what? a dictionary?

```suggestion
    def get_sub_c(self, command_name: string, command_model: dict) -> dict[]:
```

Reply:
Could you add some doc string? - I find it hard to understand what this is doing. I understand what it is doing getting a list of subcommands but the intent is not immediately obvious